_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+  Regarding a [fee overpayment attack][news101 fee overpayment attack] on segwit
+  transactions with multiple inputs, justinmoon asks why the remedy of the
+  attack, requiring copies of previous transactions for the inputs, is not
+  vulnerable to malicious software providing fake previous transactions. Since
+  any provided previous transaction must have a hash that matches the spending input’s
+  previous transaction hash, such an attack is not feasible.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta] this release candidate for an
+  LND maintenance release is now available for testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19260][] p2p: disconnect peers that send filterclear + update existing filter msg disconnect logic FIXME:dongcarl
+
+- [Bitcoin Core #19133][] rpc, cli, test: add bitcoin-cli -generate command FIXME:moneyball
+
+- [Bitcoin Core #18027][] adds two options to the GUI's *File* menu for
+  working with Partially-Signed Bitcoin Transactions ([PSBTs][topic
+  psbt]): *Load PSBT from file* and *Load PSBT from clipboard*.  When
+  one of those options is clicked and a PSBT is loaded, a dialog is
+  provided that allows the user to sign an incomplete PSBT if this
+  wallet has its key, broadcast a complete PSBT, or copy or save the
+  PSBT for processing with another tool such as [HWI][].  Combined with
+  other recent PSBT-related GUI changes (see Newsletters [#74][news74
+  psbtgui] and [#82][news82 psbtgui]) and [HWI's own GUI][hwi gui] this
+  makes it possible to use a PSBT-based process with Bitcoin Core for
+  the first time without using any RPCs.
+
+    ![Screenshot of PSBT dialog in Bitcoin Core GUI](/img/posts/2020-06-psbt-dialog.png)
+
+- [Bitcoin Core #16377][] updates the `walletcreatefundedpsbt` and
+  `fundrawtransaction` RPCs.  These RPCs normally use the wallet to
+  automatically choose which UTXOs to spend in a unsigned transaction,
+  but they also allow the user to specify one or more UTXOs they want to
+  spend in that transaction.  Previously, if the UTXOs selected by the
+  user weren't enough to pay for all of the transaction's outputs, the
+  wallet would automatically select more UTXOs to spend.  But, if a
+  user is manually selecting UTXOs, they may have some reason they don't
+  want to spend additional UTXOs, the the RPCs will now fail by default
```

Comment:
```suggestion
  want to spend additional UTXOs, the RPCs will now fail by default
```

Reply:
s/the the/and the/ (or another way to link the phrases)